beetsplug/discogs.py

```diff
@@ -351,7 +356,7 @@ def get_tracks(self, tracklist):
             # are the track index, not the medium.
             medium_is_index = track.medium and not track.medium_index and (
                 len(track.medium) != 1 or
-                ord(track.medium) - 64 != medium_count + 1
+                ord(track.medium) - 64 != medium_count * sides_per_medium + side_count
```

Comment:
This looks right to me, but perhaps an additional comment (or intermediate variable name) would make the logic more immediately obvious for future maintainability.